### PR TITLE
Remove element not found notification

### PR DESCRIPTION
**Pull request by dpgraham**
* Notification was kind of vague
* Replaced it with alert on the right panel that tells the user when they can't do the basic interactions with an element

### CODE REVIEW

File: app/renderer/components/Inspector/SelectedElement.js [hunk 1]
```diff
@@ -83,11 +83,18 @@ export default class SelectedElement extends Component {
       });
     }
 
+    const showInteractionsLoadingIndicators = !elementInteractionsNotAvailable && !elementId;
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/SelectedElement.js]: this condition is not very clear to me

File: app/renderer/components/Inspector/SelectedElement.js [hunk 2]
```diff
@@ -83,11 +83,18 @@ export default class SelectedElement extends Component {
...
+
     return <div>
+      {elementInteractionsNotAvailable && <Row type="flex" gutter={10}> 
+        <Col>
+          <Alert type="info" message="Interactions not available for this element" showIcon />
```
Comment (mykola-mokhnach) [inline app/renderer/components/Inspector/SelectedElement.js]: perhaps `are not available`?

Comment (dpgraham) [reply app/renderer/components/Inspector/SelectedElement.js]: Yeah that's probably more clear.

File: app/renderer/components/Inspector/SelectedElement.js (hunk 1 above)
Comment (dpgraham) [reply app/renderer/components/Inspector/SelectedElement.js]: Yeah it was unnecessary to have it outside of the JSX, fix pushed.

**Pull request merged**